src/_data/stats.json

```
@@ -8,6 +8,6 @@
     "lastCommitDate": "2025-12-12T22:45:02Z",
     "projectDependents": 27356303,
     "weeklyDownloads": 71087748,
-    "nextVersion": "v10.0.0-alpha.0",
+    "nextVersion": "v10.0.0-rc.0",
     "nextVersionDate": "2025-12-26"
```

```suggestion
    "nextVersionDate": "2026-01-09"
```

(or we can let it be automatically updated tomorrow)